core/controllers/dashboard_test.py

```diff
@@ -66,6 +75,349 @@ def test_logged_in_notifications_dashboard(self):
         self.logout()
 
 
+class DashboardStatisticsTest(test_utils.GenericTestBase):
+    # The model reads in each of these tests inside this class will not be
```

Comment:
I don't think you need this comment. It's relevant "in the moment", but probably won't make a lot of sense to a reader after several months, since there isn't a general presumption that model reads need to be replaced.